test/UnitTest/Mocks/Repository/InstanceRepositoryMock.cs

```diff
@@ -154,16 +154,16 @@ public Task<(Instance Instance, long InternalId)> GetOne(int instanceOwnerPartyI
             return Task.FromResult<(Instance, long)>((null, 0));
         }
 
-        public Task<Instance> Update(Instance instance)
+        public Task<Instance> Update(Instance item)
```

Comment:
This change seems counter intuitive.

Reply:
It's a complaint from the analyzers because the variable name is "item" in the interface. I can change the interface in the next iteration.